src/styles/_catalog.scss

```
@@ -4,6 +4,7 @@
 
   max-width: 944px;
   margin: 0 auto;
+  perspective: 1000px;
```

Без цього також буде працювати
```suggestion
  max-width: 944px;
  margin: 0 auto;
  perspective: 1000px;
```

---

это я тестил и забыл удалить)

---

you should probably remove this.